ci_framework/roles/manage_secrets/README.md

```diff
@@ -0,0 +1,44 @@
+# manage_secrets
+Copy secret content such as pull-secret and ci-token to a (not really) secure
+location with appropriate rights.
+
+## Privilege escalation
+None
+
+## Parameters
+* `cifmw_manage_secrets_basedir`: (String) Base directory for the directory tree. Default to `cifmw_basedir` which defaults to `~/ci-framework-data`.
+* `cifmw_manage_secrets_pullsecret_dest`: (String) Destination file for pull-secret.json. Defaults to `{{ cifmw_manage_secrets_basedir }}/secrets/pull_secret.json`.
+* `cifmw_manage_secrets_pullsecret_file`: (String) Absolute path to the source pull-secret. Defaults to `null`.
+* `cifmw_manage_secrets_pullsecret_content`: (String) Content of the source pull-secret. Defaults to `null`.
+* `cifmw_manage_secrets_citoken_dest`: (String) Destination file for ci-token. Defaults to `{{ cifmw_manage_secrets_basedir }}/secrets/ci_token`.
```

Comment:
what is a ci-token?

Reply:
something we can't really document upstream - but I mention the link to get it elsewhere in the doc